backend/helper_zero/views/auth.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+
+from helper_zero.models import Organization, DonationRequest
+from helper_zero.serializers import OrganizationSerializer, DonationRequestSerializer
+
+class AuthView(viewsets.ViewSet):
+  def list(self, request):
+    auth_token = request.GET.get('auth_token')
+    org = Organization.objects.filter(auth_token=auth_token).first()
```

Comment:
Why `.first()`? There should never be duplicate auth tokens right? We might even want to check that there aren't multiple entries returned because that should be an error

Reply:
just so that I don't get a list in return and i don't have to do like orgs[0] to get it and i'm lazy and it's a hackathon